app/gui/src/dashboard/pages/dashboard/DashboardTabBar.tsx

```diff
@@ -43,47 +45,56 @@ export function DashboardTabBar(props: DashboardTabBarProps) {
     setPage('drive')
   })
 
-  const tabs: readonly ((ProjectTabProps & { type: 'project' }) | (TabProps & { type: 'tab' }))[] =
-    [
-      {
-        type: 'tab',
-        id: 'drive',
-        icon: DriveIcon,
-        'data-testid': 'drive-tab-button',
-        labelId: 'drivePageName' satisfies TextId,
-        isActive: page === 'drive',
-        children: getText('drivePageName'),
-      },
-      ...launchedProjects.map(
-        (project) =>
-          ({
-            type: 'project',
-            id: project.id,
-            icon: NetworkIcon,
-            'data-testid': 'editor-tab-button',
-            labelId: 'editorPageName' satisfies TextId,
-            isActive: page === project.id,
-            children: project.title,
-            project,
-            onClose,
-            onLoadEnd,
-          }) as const,
-      ),
-      {
-        type: 'tab',
-        id: 'settings',
-        icon: SettingsIcon,
-        labelId: 'settingsPageName' satisfies TextId,
-        'data-testid': 'settings-tab-button',
-        isActive: true,
-        isHidden: page !== 'settings',
-        children: getText('settingsPageName'),
-        onClose: onCloseSettings,
-      },
-    ]
+  const tabs: readonly (
+    | { type: 'custom'; id: string; element: JSX.Element }
+    | (ProjectTabProps & { type: 'project' })
+    | (TabProps & { type: 'tab' })
+  )[] = [
+    {
+      type: 'tab',
+      id: 'drive',
+      icon: DriveIcon,
+      'data-testid': 'drive-tab-button',
+      labelId: 'drivePageName' satisfies TextId,
+      isActive: page === 'drive',
+      children: getText('drivePageName'),
+    },
+    ...launchedProjects.map(
+      (project) =>
+        ({
+          type: 'project',
+          id: project.id,
+          icon: NetworkIcon,
+          'data-testid': 'editor-tab-button',
+          labelId: 'editorPageName' satisfies TextId,
+          // This is fine, `page` may also be a `ProjectId`.
+          isActive: page === project.id,
+          children: project.title,
+          project,
+          onClose,
+          onLoadEnd,
+        }) as const,
+    ),
+    {
+      type: 'tab',
+      id: 'settings',
+      icon: SettingsIcon,
+      labelId: 'settingsPageName' satisfies TextId,
+      'data-testid': 'settings-tab-button',
+      isActive: true,
+      isHidden: page !== 'settings',
+      children: getText('settingsPageName'),
+      onClose: onCloseSettings,
+    },
+    {
+      type: 'custom',
+      id: 'user-bar',
+      element: userBar,
+    },
+  ]
 
   return (
-    <TabBar className="bg-primary/10" items={tabs}>
+    <TabBar className="bg-primary/5" items={tabs}>
```

Comment:
we changed it to /5, why? By design it's /10

Reply:
dunno :sweat_smile: will fix